shell/packages/sandstorm-ui-grainlist/grainlist.html

```diff
@@ -4,7 +4,7 @@
   <div class="grain-list">
     <div class="top-row">
     <div class="non-usage-info">
-    <h1>Grains</h1>
+    <h1>Grains <button class="question-mark"></button></h1>
```

Comment:
+1, I went with "What is a grain?" which is verbose but I think pretty OK. Happy to hear other ideas.